File: Step 1/Step 1.2/pattern3/pattern3.cpp
#include <iostream>
using namespace std;

int main()
{

    int n;
    cout << "Enter the number of rows ";
    cin >> n;
    int i, j;
    for (i = 1; i <= n; i++)
    {
        for (j = 1; j <= i; j++)
        {
            cout << j << " ";
        }
        cout << endl;
    }
}

File: Step 1/Step 1.2/pattern3/pattern3.java
import java.util.*;

public class pattern3 {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        int n;
        System.out.print("Enter the number of rows ");
        n = sc.nextInt();
        int i, j;
        for (i = 1; i <= n; i++) {
            for (j = 1; j <= i; j++) {
                System.out.print(j+" ");
            }
            System.out.println();
        }

    }
}